cuqi/geometry/_geometry.py

```diff
@@ -703,31 +703,49 @@ def _compute_eigpairs(self, xnod, C_nu, sigma, M, N_GL):
 class StepExpansion(Continuous1D):
     '''
     class representation of the step random field with 3 intermidiate steps
+
+    Parameters:
+    -----------
+    grid: ndarray
+        grid points
+
+    nsteps: int
+        number of equidistant steps
+
+    axis_labels: list of strings
+       plotting axis labels
```

Comment:
I dont think we need to expose this? We can just use to default from Continuous1D right?

Reply:
Good point! I removed it.